ConnectivityTestScriptsForWindows/Publish-Report.ps1

```diff
@@ -65,8 +78,16 @@ $AzureBackupCommonIssue =
 <ul>
 <li> <b>Azure Workload backup doesn't support authenticated Proxy servers. <a href ='https://docs.microsoft.com/azure/backup/backup-sql-server-database-azure-vms#use-an-http-proxy-server-to-route-traffic'> Use an HTTP proxy server to route traffic</a> </b></li>
 <li> Azure Workload Backup only support TLS 1.2 . Change TLS registry settings as described <a href = 'https://docs.microsoft.com/windows-server/security/tls/tls-registry-settings'>here</a> to enable TLS 1.2 .</li>
```

Comment:
supports

Reply:
remove space before the full stop everywhere

Reply:
Removed